src/app/queries/types/common.types.ts

```diff
@@ -4,6 +4,13 @@ export interface IMetaTypeMeta {
   kind: string;
 }
 
+interface objectReference {
```

Comment:
Just for consistency, can we stick with the capitalized interface names starting with `I`?
`IObjectReference`

Reply:
Of course!